google/cloud/bigquery/client.py

```diff
@@ -593,7 +593,7 @@ def _ensure_bqstorage_client(
             )
             return None
 
-        if bqstorage_client is None:
+        if bqstorage_client is None:  # pragma: NO COVER
```

Comment:
Looks like this branch isn't really needed anymore due to `raise_if_error=True` above.

Reply:
@tswast 
Maybe I don't understand the logic here.
If `bigquery_storage` can't be imported, we raise one of two errors and we return from the `_ensure_bqstorage_client()` function with `None`.

But if we successfully import `bigquery_storage` AND if `bqstorage_client` is `None`, we then create a new `bqstorage_client`.

What am I missing, why would we remove that branch?

Reply:
If we get to this line, we know bigquery_storage was imported

Reply:
So we don't need the if statement

Reply:
Ah yes, you're right. I misread.